bucket/xdman-beta.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "8.0.12",
+    "homepage": "https://xtremedownloadmanager.com/",
+    "description": "Beta builds of Xtreme Download Manager.",
+    "license": "GPL-3.0",

```

Comment:
Perhaps I am looking in the wrong place, but 
https://github.com/subhra74/xdm/blob/feature/8.0.1/LICENSE
shows this is still `GPL-2.0-only`, as version 7 [is](https://github.com/ScoopInstaller/Extras/blob/266dbcec884b09445f9c5ceb03bcdb873820fa8d/bucket/xdman.json#L5). 

The file
https://github.com/subhra74/xdm-experimental-binaries/blob/main/LICENSE
appears to be the license for the readme to that [repository](https://github.com/subhra74/xdm-experimental-binaries), not to the underlying source code's [repo](https://github.com/subhra74/xdm/blob/master/LICENSE).

Reply:
Hey Rasa, do you mind changing it for me please? I'm currently busy at the moment.

Reply:
Done.